legate/driver/args.py

```diff
@@ -99,12 +100,39 @@ def _get_mv2_config() -> tuple[int, int] | None:
     return ranks // ranks_per_node, ranks_per_node
 
 
+_SLURM_CONFIG_ERROR = (
+    "Expected SLURM_TASKS_PER_NODE to be a single integer ranks per node, or "
+    "of the form 'A(xB)' where A is an integer ranks per node, and B is an "
+    "integer number of nodes, got SLURM_TASKS_PER_NODE={value}"
+)
+
+
 def _get_slurm_config() -> tuple[int, int] | None:
     if not (nodes_env := getenv("SLURM_JOB_NUM_NODES")):
         return None
```

Comment:
Should we always demand this? the new `A(xB)` case does not rely on `SLURM_JOB_NUM_NODES`

Reply:
half of the `SLURM_TASKS_PER_NODE` case does need it tho (single integer case). It seems like its always provided in any case, so I'll leave this as it is for now.